contact.md

```diff
@@ -64,9 +64,9 @@ To leave the mailing list, send an email to [listserv@listserv.llnl.gov][devunsu
 <a href="https://www.youtube.com/results?search_query=UVCDAT"><img class="contact-logo" src="/images/youtube.png" alt="YouTube logo"/></a>
 
 [uvcdat-askbot]:   http://askbot-uvcdat.llnl.gov
```

Comment:
askbot is dead, can you please remove it. Thanks.

Reply:
Done.